src/vcf/validator.cpp

```diff
@@ -145,6 +145,16 @@ namespace ebi
         }
 
         validator.end();
+        for (auto &error : validator.errors()) {
```

Comment:
it doesn't show the same behaviour to me. `vcf_validator -vv4.3 -i test/input_files/v4.3/failed/failed_body_no_newline_004.vcf` for instance, with those loops, it shows correctly "Line 3: There is no newline at the end of the file"; but without the loops, it still says that the file is invalid, but with no messages at all.